packages/spectral/src/serverTypes/convertIntegration.ts

```diff
@@ -170,6 +180,85 @@ const codeNativeIntegrationYaml = (
   return YAML.stringify(result);
 };
 
+type VisibilityAndPermissionValue = {
+  [Key in `visibleTo${Capitalize<PersonaType>}Deployer`]: boolean;
```

Comment:
This is fun but probably overly clever since it's the only place we're doing this, the only place we use `PersonaType`, and requires hopping back and forth between two files to figure out what it is.